src/blocks/weather/open_weather_map.rs

```diff
@@ -388,8 +388,8 @@ impl WeatherProvider for Service<'_> {
             })
         };
 
-        let sunrise = Some(DateTime::<Utc>::from_timestamp(current_data.sys.sunrise, 0).unwrap());
-        let sunset = Some(DateTime::<Utc>::from_timestamp(current_data.sys.sunset, 0).unwrap());
+        let sunrise = DateTime::<Utc>::from_timestamp(current_data.sys.sunrise, 0);
```

Comment:
@MaxVerevkin suggested doing this, so that if there is an error it will be surfaced.

```rs
let sunrise = Some(DateTime::<Utc>::from_timestamp(current_data.sys.sunrise, 0).error("Unable to convert timestamp to DateTime")?);
```